README.md

```diff
@@ -27,6 +27,8 @@ To build Spark and its example programs, run:
 (You do not need to do this if you downloaded a pre-built package.)
 More detailed documentation is available from the project site, at
 ["Building Spark"](http://spark.apache.org/docs/latest/building-spark.html).
+For developing Spark using an IDE, see [Eclipse](https://cwiki.apache.org/confluence/display/SPARK/Useful+Developer+Tools#UsefulDeveloperTools-Eclipse)
```

Comment:
I agree as I won't be the one maintaining the documentation.
Do you want a single link or are two links also ok for you?